### ISSUE TITLE
Update the styling of the background

### ISSUE THREAD

**Issue by rohitja26** [rohitja26/MERN-Pickpot#1]
The current background styling of the project could be enhanced for a more modern and visually appealing design. We're looking for contributions to update the background styling across the app to make it more consistent and user-friendly. Some potential improvements could include:

- Adding a gradient or image-based background
- Improving color schemes to align with the overall theme of the app
- Ensuring responsive design for different screen sizes and devices
- Following good contrast and readability practices for text over the background

Feel free to get creative while keeping the UI clean and professional. All changes should be documented, and any external libraries or resources used should be properly referenced. Contributions are welcome from all levels of experience!

**Comment by huzaifsk** [comment]
@rohitja26 can I help you with styling please

**Comment by rohitja26** [comment]
Yes, @huzaifsk  please star ⭐the repo.  And fork. After that start working on this

**Comment by huzaifsk** [comment]
@rohitja26  please review my PR
